website/docs/src/content/quickstart.mdx

```diff
@@ -341,10 +341,233 @@ const stream = await workflow.run({ topic: "latest quantum computing chips" });
 
 ![blog writing trace](/quickstart/blog-trace.png)
 
-## Next steps
+## Running the dev server
 
-Now that you've gone through the quickstart, you should be able to start building with GenSX. Take a look at the following examples to see how you can build more complex workflows.
+GenSX provides a local development with local REST APIs that match the shape of workflows deployed to GenSX Cloud. Run the dev server from the CLI:
```

Comment:
```suggestion
GenSX provides a local development server with local REST APIs that match the shape of workflows deployed to GenSX Cloud. You can run the dev server from the CLI:
```

Reply:
We should also add in a sentence before this like "Now that you've built your workflows, you can easily turn them into REST APIs."

Keeps the tutorial/quickstart type vibe going.